### PR TITLE
feat(sbb-sticky-bar): fade in vertically during initial load

### LINKED ISSUE #3072: [Feature request]: Transition of the sticky bar when triggered by an action
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/orgs/sbb-design-systems/projects/4) for an issue that matches the one I want to file, without success.

### Request type

Request for enhancement of a component

### It affects the following packages

Components

### Functionality

An option is required in the sticky bar so that the sticky bar can move up from below when it is triggered by an action on the website. The aim is to have at least the same duration as in the time table when level 2 appears after a click on level 1.

Here's an example (Figma prototype: Klick on the 1st level connection to open the 2nd level and the sticky bar will appear):
https://www.figma.com/proto/0pGd1tS4HLCDUf5U1qQpxn/03%3A-Design-%E2%80%93-Timetable-results?page-id=883%3A589982&node-id=3547-865589&node-type=canvas&viewport=642%2C234%2C0.15&t=FtmwKhpG95Et5Gyp-1&scaling=scale-down&content-scaling=fixed&starting-point-node-id=883%3A589984

In the design proposal file you'll find an overview with the durations of the transition: Ease out, 160ms

### Link to design proposal file

https://www.figma.com/design/0pGd1tS4HLCDUf5U1qQpxn/03%3A-Design-%E2%80%93-Timetable-results?node-id=883-589982

### PR DESCRIPTION
**Pull request by jeripeierSBB**
Closes #3072

BEGIN_COMMIT_OVERRIDE
feat(sbb-sticky-bar): introduce controllable slide in and out animation

Closes #3072
END_COMMIT_OVERRIDE

### CODE REVIEW

File: src/elements/container/sticky-bar/sticky-bar.ts [hunk 1]
```diff
@@ -61,10 +61,27 @@ export class SbbStickyBarElement extends LitElement {
   }
 
   private _toggleShadowVisibility(entry: IntersectionObserverEntry): void {
+    const isSticky = !entry.isIntersecting && entry.boundingClientRect.top > 0;
+
+    const intersectorRect = this._intersector?.getBoundingClientRect();
+    const stickyBarRect = this.shadowRoot!.querySelector(
+      '.sbb-sticky-bar__wrapper',
+    )!.getBoundingClientRect();
+    const HEIGHT_TOLERANCE = 30;
```
Comment (MarioCastigliano) [inline src/elements/container/sticky-bar/sticky-bar.ts:70]: Would it make sense to make this configurable/overridable? Perhaps in other scenarios the behavior might change slightly and the consumer might want to fine-tune it.

**Review by MarioCastigliano — APPROVED** [review]
Just a few considerations, but overall a solid job 👍

File: src/elements/container/sticky-bar/sticky-bar.ts (hunk 1 above)
Comment (jeripeierSBB) [reply src/elements/container/sticky-bar/sticky-bar.ts:70]: Discussed it with Lukas, for now not

**Review by kyubisation — APPROVED** [review]
LGTM 👍

**Pull request merged**